src/mo_frontend/typing.ml

```diff
@@ -972,6 +972,16 @@ and infer_exp'' env exp : T.typ =
       ot := t
     end;
     T.text
+  | ToCandidE exps ->
+    if not env.pre then begin
+        let ts = List.map (infer_exp env) exps in
+        if not (T.shared (T.Tup ts)) then
```

Comment:
```suggestion
        if not (T.shared (T.seq ts)) then
```

Reply:
I get that we are uniformly making this change (and I will here), but this chance is stylistic only, right?

In other words, the two program versions are equivalent under all inputs, right?